.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Run Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  create:
+    tags:
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7
```

Comment:
Please use the Ruby version called out in the Gemfile. Please also update .ruby-version to use that same number.